Automationexercise/TestsUI/AutomationExerciseTest.cs

```diff
@@ -0,0 +1,242 @@
+        }
+
+        [Order(4)]
+        [Test]
+        [Description("Test Case 11: Verify Subscription in Cart page")]
+        public async Task VerifySubscriptionInCartPage()
+        {
+            await _common.HomePageTitle.IsVisibleAsync();
+            await _common.ClickCartButton();
+            await _common.ScrollToFooter();
+            Assert.That(await _common.SubscriptionTitle.IsVisibleAsync(), "'SUBSCRIPTION' text is not visible");
+            await _common.EnterEmailInSubscriptionInput("jondoe@mailinator.com");
+            await _common.ClickSubscriptionArrowButton();
+            Assert.That(await _common.SubscribeSuccessMessage.InnerTextAsync(), Contains.Substring("You have been successfully subscribed!"),
+                "Subscription success message is not visible");
+        }
+
+        [Order(5)]
+        [Test]
+        [Description("Test Case 13: Verify Product quantity in Cart")]
+        [TestCase("Winter Top", "4")]
+        public async Task VerifyProductQuantityInCart(string productName, string quantity)
+        {
+            await _common.HomePageTitle.IsVisibleAsync();
+            await _productsPage.ClickViewProductButton(productName);
+            await _productDetailsPage.SelectQuantity(quantity);
+            await _productDetailsPage.ClickAddToCartButton();
+            await _cart.GoShoppingButton.WaitForAsync(new LocatorWaitForOptions { State = WaitForSelectorState.Visible });
+            await _cart.ClickViewCartButton();
+            Assert.That(await _cart.GetProductQuantity(productName), Is.EqualTo(quantity), "Product quantity is not visible");
+        }
+        
+        [Order(6)]
+        [Test]
+        [Description("Test Case 16: Place Order: Login before Checkout")]
+        [TestCase("Blue Top", "1")]
+        public async Task PlaceOrderLoginBeforeCheckout(string productName, string quantity)
+        {
+            await _common.HomePageTitle.IsVisibleAsync();
+            await _cart.AddToCart(productName);
+            await _cart.ClickViewCartButton();
+            Assert.That(await _cart.IsCartPageVisible(), Is.True,"Cart page is not visible");
+            await _cart.ClickProceedToCheckoutButton();
+            Assert.That(await _checkoutPage.IsAddressDetailsVisible(_user), Is.True,"Address details is not visible");
+            Assert.That(await _checkoutPage.IsReviewYourOrderVisible(productName, quantity), Is.True,"Review your order is not visible");
+            await _checkoutPage.EnterDescriptionInCommentField("This is a test order");
+            await _checkoutPage.ClickPlaceOrderButton();
+            await _paymentPage.EnterPaymentDetails("John Doe", "1234567890123456", "123", "12","25");
+            await _paymentPage.ClickPayAndConfirmOrderButton();
+            Assert.That(await Page.GetByText("Congratulations! Your order has been confirmed!").IsVisibleAsync(), "Order Placed success message is not visible");
+            await _common.ClickDeleteAccountButton();
```

Comment:
delete account is not needed here. User is removed in OneTimeTearDown

Reply:
removed